scania-digital-design-system/sdds
# feat(issue templates): update issue templates for bug, feature and question

**Describe pull-request**  
_Describe what the pull-request is about_

**Solving issue**  
_In case of GitHub issue add # plus the number of the issue (for example #123) OR if it is Azure then AB# and number of ticket_
Fixes: [AB#1293](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1293)

**How to test**  
_Add description how to test if possible_
1. Go to .github/issue_templates folder or [here](https://github.com/mmexvr/sdds/tree/feature/new_issue_templates/.github/ISSUE_TEMPLATE)
2. Check in all the new templates
3. Are there any problems with spelling or the questions
